__tests__/date.test.js

```diff
@@ -0,0 +1,9 @@
+const dateService = require("../services/date");
```

Comment:
rename the file to zodiacsign-test, something like this. File name should reflect what test it does